src/static_assert.h

```diff
@@ -0,0 +1,27 @@
+/* Pi-hole: A black hole for Internet advertisements
+*  (c) 2021 Pi-hole, LLC (https://pi-hole.net)
+*  Network-wide ad blocking via your own hardware.
+*
+*  FTL Engine
+*  Struct size assertion tool
+*
+*  This file is copyright under the latest version of the EUPL.
+*  Please see LICENSE file for your rights under this license. */
+
+#include <assert.h>
+
+
+#define STATIC_ASSERT(OBJECT, EXPECTED)   \
+  static_assert(sizeof(OBJECT) == EXPECTED , "Expected size of " #OBJECT " is " #EXPECTED " on this architecture.");
```

Comment:
Is there a way to have the `sizeof(OBJECT)` included in the warning message? It only tells me what the expected size is, I don't know what the actual size was.